### PR TITLE
Refactor the LN backend and add more logs

### PR DESCRIPTION
**Pull request by nymkappa**
This PR cleans up a bit the lightning backend, mostly the node and channel updater code.
I've also added logs so we get a progress update every 10 seconds approximately.

We run the node/channels update every `config.MEMPOOL.GRAPH_REFRESH_INTERVAL` now (default to 5 minutes).

Creates and fixes https://github.com/mempool/mempool/issues/2256

### CODE REVIEW

File: backend/src/api/explorer/nodes.api.ts [hunk 1]
```diff
@@ -321,6 +323,66 @@ class NodesApi {
...
+      ]);
+    } catch (e) {
+      logger.err('$saveNode() error: ' + (e instanceof Error ? e.message : e));
+    }
+  }
+
+  /**
+   * Set all nodes not in `nodesPubkeys` as inactive (status = 0)
+   */
+   public async $setNodesInactive(graphNodesPubkeys: string[]): Promise<void> {
```
Comment (nymkappa) [inline backend/src/api/explorer/nodes.api.ts:364]: This function `$setNodesInactive` is not yet used, you can ignore it for this PR.

File: backend/src/config.ts [hunk 2]
```diff
@@ -32,7 +32,8 @@ interface IConfig {
     ENABLED: boolean;
     BACKEND: 'lnd' | 'cln' | 'ldk';
     TOPOLOGY_FOLDER: string;
-    NODE_STATS_REFRESH_INTERVAL: number;
+    STATS_REFRESH_INTERVAL: number;
```
Comment (nymkappa) [inline backend/src/config.ts:35]: Renamed `NODE_STATS_...` to `NETWORK_STATS_...` to avoid any future confusion

**Comment by nymkappa** [comment]
There may be an issue right now, I'm checking. Related to https://github.com/mempool/mempool/issues/2256.
Seems to work fine on master, so this PR may have an issue.

```
Aug  8 10:28:26 [90104] INFO: <lightning> Importing historical lightning stats
/Users/nymkappa/Documents/mempool/backend/dist/rpc-api/jsonrpc.js:128
                            err = new Error(decodedResponse.error.message || '');
                                  ^

Error: JSON integer out of range
    at /Users/nymkappa/Documents/mempool/backend/dist/rpc-api/jsonrpc.js:128:35
    at Array.forEach (<anonymous>)
    at IncomingMessage.<anonymous> (/Users/nymkappa/Documents/mempool/backend/dist/rpc-api/jsonrpc.js:125:25)
    at IncomingMessage.emit (node:events:539:35)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: -1
}
```

**Comment by nymkappa** [comment]
Creates and fixes https://github.com/mempool/mempool/issues/2256

**Review by wiz — CHANGES_REQUESTED** [review]
Tested NACK @ ef6c380
```
Aug 10 12:07:15 node201.fra  mempool[92168]: INFO <testnet> Fetching hourly price history from exchanges and saving missing ones into the database, this may take a while
Aug 10 12:07:15 node201.fra  mempool[91346]: ERR <lightning> $saveNode() error: Unknown column 'status' in 'field list'
Aug 10 12:07:15 node201 syslogd: last message repeated 300 times
Aug 10 12:07:15 node201.fra  mempool[90767]: NOTICE Inserted 1 daily historical prices into the db
Aug 10 12:07:15 node201.fra  mempool[91346]: ERR <lightning> $saveNode() error: Unknown column 'status' in 'field list'
Aug 10 12:07:16 node201 syslogd: last message repeated 52 times
Aug 10 12:07:16 node201.fra  mempool[90767]: INFO Fetching hourly price history from exchanges and saving missing ones into the database, this may take a while
Aug 10 12:07:16 node201.fra  mempool[91346]: ERR <lightning> $saveNode() error: Unknown column 'status' in 'field list'
Aug 10 12:07:17 node201 syslogd: last message repeated 9078 times
```

**Comment by wiz** [comment]
Replaced by #2267

**Pull request closed without merging**